crates/gpui/src/platform/linux/x11/window.rs

```diff
@@ -202,13 +202,22 @@ impl X11WindowState {
             })
             .unwrap();
 
-        // Is this ID used anywhere?
-        let idle_notify_event_id = xcb_connection.generate_id();
+        // Selects the set of Present events to be delivered for the specified window and event context.
         xcb_connection
             .send_and_check_request(&xcb::present::SelectInput {
-                eid: idle_notify_event_id,
+                eid: xcb_connection.generate_id(),
                 window: x_window,
-                event_mask: xcb::present::EventMask::IDLE_NOTIFY,
+                event_mask: xcb::present::EventMask::COMPLETE_NOTIFY,
```

Comment:
is there a reason IDLE_NOTIFY can't be used?

Reply:
> IdleNotify events are delivered when a pixmap used in a PresentPixmap operation may be re-used by the client.

Seeing as we don't use a pixmap directly (I assume this is handled in the vulkan driver) we cannot call `PresentPixmap` so we will never get this event.

Whereas for complete notify:
> CompleteNotify events are delivered when a PresentPixmap or PresentNotifyMSC operation has completed.